eZ/Publish/SPI/Search/FullTextFieldValue.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+/**
+ * This file is part of the eZ Publish Kernel package.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace eZ\Publish\SPI\Search;
+
+/**
+ * The wrapper for \eZ\Publish\SPI\Search\Field of the type \eZ\Publish\SPI\Search\FieldType\FullTextField.
+ */
+class FullTextFieldValue extends Field
+{
+    /**
+     * @param mixed $value
+     */
+    public function __construct($value)
+    {
+        parent::__construct('fulltext', (string) $value, new FieldType\FullTextField());
```

Comment:
@pspanja: Sorry, I forgot to mention that I had to go back to returning `Search\Field[]` from `getFullTextData`, so it is in sync with `getFilterData` - for mappers to use them in a generic way. 
Usage is shown for ES in f435c88.

It allows me not to repeat 
`new Search\Field('fulltext', $value, new FieldType\FullTextField())` 
for each field (9391fa4).
> Also, since this is API (SPI), I would do this kind of thing in the implementation and not here.

I agree, API/SPI is probably not a good place for this, however I would like to have it somewhere common for all search engines implementations (`eZ\Publish\Core\Search\Common`?).
But if you still feel it's bad, I'll change it back to `Search\Field` :)